### PR TITLE
Add Next.js and Remix.js examples

### PR DESCRIPTION
**Pull request by decepulis**
- For Next.js, we just link to `vercel/next.js/examples`
- For Remix, I'd _like_ to link to `remix-run/examples`, but that PR hasn't been accepted yet. (Nor does it look like it'll be accepted any time soon; that repo is kinda a ghost town)
    - So for now, I'm hosting the example here, with a note at the top of the README. If the remix PR is ever accepted, I'll remove the example here and link to the example there just like I do with next.js
    - I also considered just linking to the remix-run/examples PR... So instead of hosting the whole example, we'd just have a link in the readme [to this](https://github.com/remix-run/examples/pull/441). lmk if you think that would be more consistent

### CODE REVIEW

File: remixjs-uploader-and-player/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/examples/tree/main/mux-video)
+
+## How to use
+
+### Step 1. Create a Remix app with this example
+
+```bash
+npx create-remix@latest --template examples/mux-video
```
Comment (davekiss) [inline remixjs-uploader-and-player/README.md:25]: Will this work if the external PR is not accepted?

File: remixjs-uploader-and-player/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+These steps correspond to the following routes:
+
+1. [`_index.tsx`](app/routes/_index.tsx) creates the upload in a loader, and exchanges the `upload.id` for an `asset.id` in an action which redirects to...
+2. [`status.$assetId.tsx`](app/routes/status.$assetId.tsx) polls the Mux API to see if the asset is ready. When it is, we redirect to...
+3. [`playback.$playbackId.tsx`](app/routes/playback.$playbackId.tsx) plays the video.
+
+## Preparing for Production
+
+### Set the cors_origin
```
Comment (davekiss) [inline remixjs-uploader-and-player/README.md:66]:
```suggestion
### Set the `cors_origin`
```

**Review by davekiss — APPROVED** [review]
Approving, only one comment re: cloning the template

File: remixjs-uploader-and-player/README.md (hunk 1 above)
Comment (decepulis) [reply remixjs-uploader-and-player/README.md:25]: good callout. gotta fix that.

**Pull request merged**